parsel/selector.py

```diff
@@ -90,9 +90,15 @@ def create_root_node(
     parser_cls: Type[_ParserType],
     base_url: Optional[str] = None,
     huge_tree: bool = LXML_SUPPORTS_HUGE_TREE,
+    body: Optional[bytes] = "",
+    encoding: Optional[str] = "utf8",
```

Comment:
I do not see a reason to allow `None` as an encoding here either.

```suggestion
    encoding: str = "utf8",
```

But if we do, maybe we should make `None` the default value instead of `"utf8"`, and use it when encoding `text` into `body`.

Reply:
done